### DISCUSSION TITLE
Wrapping an image around a 3D object

### DISCUSSION THREAD

**Discussion by ivanamar996** [dlemstra/Magick.NET#1384] [General]
Hi,

Is it possible to wrap an image on previously created 3D object by using Magick.NET library? In order to achieve something similar to https://im.snibgo.com/pntppl.htm#Scripts:~:text=repage%20%5E%0A%20%20pntp_newskin.miff-,Method%202%3A%20prepare%20in%203D%2C%20then%20operate%20in%202D,-The%20second%20method.

Kind regards,
Ivana Markovic

**Comment by dlemstra** [comment]
It is probably possible but I don't know if there are any examples available to do this.
